stdlib/public/Platform/Platform.swift

```diff
@@ -103,8 +103,7 @@ public var errno : Int32 {
   get {
 #if os(OSX) || os(iOS) || os(watchOS) || os(tvOS) || os(FreeBSD)
     return __error().pointee
-//FIXME: os(Windows) should be replaced, such as triple(Cygwin)
```

Comment:
There was a discussion about using ```os(Cygwin)``` or introducing new condition ```env()```. I agree that Cygwin is not OS nor OS emulator, but I think using ```os()``` is practically not bad for the present.

https://lists.swift.org/pipermail/swift-dev/Week-of-Mon-20160425/001853.html
https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20160425/015852.html